test/cmp_testlib.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2007-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright Nokia 2007-2018
+ * Copyright Siemens AG 2015-2018
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ * Copyright OpenSSL 2007-2018
```

Comment:
You are repeating things here in the license text ... needs to be cleaned up.

Reply:
Fixed